cookbook/integration_huggingface_openai_sdk.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<p>what is <code>tgi</code> that is referenced in the following examples, maybe add a small note on this  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/1154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>